sechub-schedule/src/main/java/com/daimler/sechub/domain/schedule/access/ScheduleAccessRepository.java

```diff
@@ -11,9 +11,12 @@
 
 public interface ScheduleAccessRepository extends JpaRepository<ScheduleAccess, ProjectAccessCompositeKey> {
 
+	@Query(value="SELECT EXISTS(SELECT * FROM "+TABLE_NAME+" where "+COLUMN_PROJECT_ID+" = ?1)",nativeQuery=true)
```

Comment:
I know this is done also at other query inside this class, but in future please try always to use spring data annotation `@Param("myIdentifier") and use it inside the query by " =:myIdentifier"

Its okay for one parameter, but having multiple ones named parameters are much better because no ordering problems when query changes - its also more readable

Reply:
done